README.md

```diff
@@ -1,16 +1,192 @@
-# Title
+# Arrow Function Shorthand
+
+## Introduction
+
+The original style for declaring functions in JavaScript is the _function
+declaration_.
+
+```js
+function foo() {
+  return 'bar';
+}
+```
+
+But JavaScript has two other ways to write functions: the _function expression_
+and the _arrow function_. None of these is more correct or better than others.
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Declare a function using a function expression
+- Declare a function using an arrow function
+- Describe situations where arrow functions are used
 
-## Introduction
+## Declare a Function Using a Function Expression
+
+Thus far we've only ever used a _function declaration_:
+
+```js
+function foo() {
+  return 'bar';
+}
+```
+
+A function can also be written:
+
+```js
+let foo = function() {
+  return 'bar';
+}
+```
+
+The `function() {...}` to the right of the assignment operator (`=`) is called
+a _function expression_. The best way to understand what a function expression
+is is by analogy.
+
+```js
+let sum = 1 + 1
+```
+
+Evaluate the expression `1 + 1`, returning `2` and assign it to the variable `sum`
+
+```js
+let difference = 10 - 1;
+```
+
+Evaluate the expression `10 - 1`, returning `9` and assign it to the variable
+`difference`.
+
+```js
+let foo = function() {
+  return 'bar';
+}
+```
+
+Evaluate the expression `function() { return 'bar' }`, returning a thing that
+can be called and assign it to the variable `foo`. The function expression  
+(again, the thing to the right of `=`) is known as "an anonymous
+function." It doesn't have a name associated with it like you see in a
+_function declaration_.
+
+However, when we assign an anonymous function to a name (that is, a variable),
+we have name that points to a callable thing. We could call this anonymous
+function by invoking `foo()`. That anonymous function is now, for all
+reasonable purposes, named `foo`.
+
+There are a few subtle differences between _function declarations_ and
+_function expressions_, but they are very minute. Neither is really better than
+the other. JavaScript supports variety. Neither is better or more preferred
+than the other.
```

Comment:
Anything that ends with a "neither is better" sentiment makes me want to hear more. I think from a student's perspective, it's frustrating to be told these different things are equal and not have a clear idea of why I need to know about all of them. It's helpful to close this with an assurance that recognition is useful or one is more common or something similar, or to set up the next section.